nibabies/workflows/anatomical/brain_extraction.py

```diff
@@ -282,7 +282,8 @@ def init_precomputed_mask_wf(
         name="outputnode",
     )
 
-    validate_mask = pe.Node(ValidateImage(), name="validate_mask")
+    # no need for ValidateImage if T1w header will be copied
```

Comment:
What if we reorient the incoming T1w image, but the mask is in the original orientation?